public/app/features/alerting/unified/RuleEditorGrafanaRules.test.tsx

```diff
@@ -70,4 +70,30 @@ describe('RuleEditor grafana managed rules', () => {
 
     expect(await screen.findByRole('status')).toHaveTextContent('Rule added successfully');
   });
+
+  it.skip('shows an error when trying to use time series as alert condition', async () => {
```

Comment:
I couldn't get this test to work all the way yet - it was also a bit fiddly to debug why it wasn't showing the alert. The test all works except the last assertion - will try and continue working out why this is broken